Looms.PoS/Looms.PoS.Application/Mappings/ModelsResolvers/BusinessModelsResolver.cs

```diff
@@ -18,6 +18,23 @@ public BusinessModelsResolver(IMapper mapper)
     public BusinessDao GetDaoFromRequest(CreateBusinessRequest createBusinessRequest)
         => _mapper.Map<BusinessDao>(createBusinessRequest);
 
+    public BusinessDao GetDaoFromDaoAndRequest(BusinessDao originalDao, UpdateBusinessRequest updateBusinessRequest)
+    {
+        return _mapper.Map<BusinessDao>(updateBusinessRequest) with
+        {
+            Id = originalDao.Id,
+            OwnerName = originalDao.OwnerName,
+        };
+    }
+
+    public BusinessDao GetDeletedDao(BusinessDao originalDao)
+    {
+        return originalDao with
+        {
+            IsDeleted = true
+        };
+    }
```

Comment:
I believe that this should be in a different place as model resolver shouldn't contain additional logic. Could be wrong though.

Reply:
Changing property values is handled by the models resolver